### PR TITLE
Update Create Project API

### PR DESCRIPTION
**Pull request by ysmolyar**
This is part of the work that will create/delete (rotate) project tokens as the current Vault configuration sets an expiration of one year for AppRole secrets.

Adding implementation to return token_id along with the token when creating a project.

### CODE REVIEW

**Comment by pclata** [comment]
@ysmolyar can you resolve the conflicts?

File: service/test/testdata/TestCreateProject/can_create_project_response.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "token": "vault::"
+  "token": "vault::",
+  "token_id": ""
```
Comment (pclata) [inline service/test/testdata/TestCreateProject/can_create_project_response.json:3]: Would an empty token ID mean that create project failed somewhere in its flow? If so, I would expect this test to fail.

**Review by pclata — CHANGES_REQUESTED** [review]
Overall LGTM, but just one small comment.

File: service/test/testdata/TestCreateProject/can_create_project_response.json (hunk 1 above)
Comment (ysmolyar) [reply service/test/testdata/TestCreateProject/can_create_project_response.json]: sure, updated the mock to provide values here. now expecting:
`
{
  "token": "vault:role-id:secret",
  "token_id": "secret-id-accessor"
}
`

**Review by pclata — APPROVED** [review]
LGTM

**Pull request merged**